Lib/smtplib.py

```diff
@@ -933,6 +933,7 @@ def send_message(self, msg, from_addr=None, to_addrs=None,
             from_addr = (msg[header_prefix + 'Sender']
                            if (header_prefix + 'Sender') in msg
                            else msg[header_prefix + 'From'])
+            from_addr = [a[1] for a in email.utils.getaddresses([from_addr])][0]
```

Comment:
SInce there's only one, you don't need the list comprehension:

   from_addr = email.utils.getaddresses([from_addr])[0][1]

Reply:
@bitdancer about your solution, I like it and that was my first solution, but I was thinking about one thing, I didn't see in the code an section where you have this kind of access to the item `[0][1]` but if you agree with that, I am going to fix it.